### ISSUE TITLE
Error when submitting a smiley in a (Multiline) field that is part of a group.

### ISSUE THREAD

**Issue by bartdigitalpulse** [verbb/formie#774]
**Description**
Error: PDOException: SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\xF0\x9F\x98\x89.' for column 

When the enduser enters a smiley in a field that is part of a Group it will not get checked with containsMb4 StringValidator setting. This will causes a SQL error. When the field is not part of a Group it will show a correct validation error: smiley not allowed. Full stack trace as attachment.

**Steps to reproduce**
1. Create new Group
2. Add (Multi-line Text) Field to Group
3. Enter a smiley in the form on the frontend: Hi! 😉
4. On submit you should get the error

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.9
- Craft version: Craft Pro 3.7.29
- Multi-site: Yes

**Additional context**
[error.txt](https://github.com/verbb/formie/files/8116283/error.txt)

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.10](https://github.com/verbb/formie/releases/tag/1.5.10)
